### ISSUE TITLE
新版本打不开Wi-Fi

### ISSUE THREAD

**Issue by icycommond** [weachy/CoreBookX-14-Hackintosh#10]
我的是原版Intel网卡，更新1108版本后搜不到Wi-Fi，系统设置里也没有蓝牙图标

**Comment by matias200308** [comment]
tengo el mismo problema, intenté conectarme por Wi-Fi y por ethernet y no tuve suerte.

**Comment by weachy** [comment]
> 我的是原版Intel网卡，更新1108版本后搜不到Wi-Fi，系统设置里也没有蓝牙图标

提交了修补，我这边使用博通网卡，无法测试，你可以再试试看。

**Comment by icycommond** [comment]
使用新版后 还是搜不到Wi-Fi，
任务栏显示Wi-Fi是打开的，但是搜不到任何网络，
系统设置里网络显示Wi-Fi是关闭状态，点击开启没有反应，
是不是网卡id不对

**Comment by farisoftfry** [comment]
Hello, I had a similar problem with Bluetooth, after using Apple Magic Mouse (it worked). After a restart bluetooth disappeared and I wasn't able to load bluetooth settings too.
Reinstalling Monterey didn't solved the problem even if before I've deleted any partition from the SSD.
I don't know why it worked but I solved the problem deleting every partitions on the SSD, then I've installed Windows and installed every drivers with Driver Booster free. Bluetooth worked (I was worried it was broken).
Then I've deleted again every partition of the SSD and I've reinstalled Monterey with the latest EFI release from weachy and everything works now. Of course, pc does a strange sound during start or restart so maybe there is something else to set or update, I don't know, but it is a great work from weachy! thanks a lot! Maybe someone could explain why it worked, but I solved in this way.
Hope it helps.
